content/blog/2025-11-03-urssi-fall-fellowship-ai-in-sci-soft.md

```diff
@@ -0,0 +1,86 @@
+
+## Why It Matters
+
+The scientific software community has potentially entered an era where code can be rapidly prototyped, deployed, and integrated into research processes. This brings exciting opportunities for productivity and creativity, but also raises challenges for transparency, trust, and correctness.
+
+Our findings will help:
+
+1. Characterize the change in scientific software practices since the advent of LLMs.
+
+2. Inform, engage, and empower the scientific software community—maintainers, developers, and users alike—to engage critically and collaboratively with the growing role of AI-generated code.
+
+3. Encourage reproducible science by improving awareness of how AI contributions shape research codebases.
+
+All datasets, software, and models we create will be open-source, enabling others to build on this work.
+
+## How We’re Engaging the Community
+
+A central part of the project is collaboration with the research software community:
+
+1. **Open Resources:** We’ll publish our dataset, selected detection models, and analysis scripts under a permissive license, along with thorough documentation.
+
+2. **Collaborative Feedback:** We plan on inviting scientific software maintainers and computational researchers to share feedback on integration strategies, best practices, or general collaboration.
+
+3. **Workshops and Presentations:** We plan to share progress and results, focusing on practical takeaways for the community.
+
+If you’re interested in contributing, providing feedback, or testing our tools, please reach out! 
+
+## What’s Next
+
+Over the next six months, we’ll be:
+
+1. Curating our dataset of scientific repositories. 
+2. Evaluting and creating our detection tool which integrates existing models.
+3. Analyzing code quality, documentation, and maintainability trends.
+4. Presenting results at community events.
+
+## Thank You!
+
+This project is supported by a **$25,000 fellowship with the US Research Software Sustainability Institute**, with funding from the **Sloan Foundation**. The fellowship includes a research stipend and funding for computing resources, LLM API access, and travel to conferences for disseminating our findings.
+
+Without this support, our project would not be possible. Thank you for providing both the mentorship and funding that make this research possible and for helping us continue advancing the responsible use of AI in scientific software.
```

Comment:
thanks, but not necessary to include this for the broader URSSI community to read